Makefile

```diff
@@ -14,9 +14,14 @@ build:
 install:
 	python setup.py install
 
+.PHONY: test
+test:
+	py.test -s tests
+
 PYTHON_FORMAT ?= yapf
 format:
 	$(IN_ENV) git ls-files | grep -ve '^third_party\|^\.' | grep -e '.py$$' | xargs -r -P $$(nproc) yapf -p -i
+	flake8 .
```

Comment:
`make format` shouldn't need anything done manually.

I think either `flake8 .` should go under `test:` and maybe add a `make check` which does a `make format; make test` type thing?

Reply:
I've moved `flake8` under `lint:`, and added a `quick-check` target that runs all the quick checks by default when you run `make`: `format lint check format-cpp build test check-license check-python-scripts`